generators/client/templates/angular/src/main/webapp/app/account/sessions/sessions.component.ts.ejs

```diff
@@ -21,41 +21,40 @@ import { Component, OnInit } from '@angular/core';
 import { AccountService } from 'app/core/auth/account.service';
 import { Session } from './session.model';
 import { SessionsService } from './sessions.service';
+import { Account } from 'app/core/user/account.model';
 
 @Component({
     selector: '<%= jhiPrefixDashed %>-sessions',
     templateUrl: './sessions.component.html'
 })
 export class SessionsComponent implements OnInit {
-
-    account: any;
-    error: string;
-    success: string;
-    sessions: Session[];
+    account: Account | undefined;
```

Comment:
Could also write this shorter as `account?: Account`, couldn't you?

Reply:
@codecholeric Thanks for looking at that!  
`?:` seems to work here too, but as `?` means optional then it seems to me that `| undefined` fits better to context.

Reply:
But what is the difference for you, between "it's optional", and "it can be `undefined`"? And where do you distinguish with `null` :wink: (that one always gave me a headache with JS)

Reply:
I changed `| undfined` to `| null = null` in all 3 places. I believe it's better now. @codecholeric thanks for pointing this out and helping to make code cleaner!

Reply:
@codecholeric, after resolving strict option in some other modules I think your suggestion to use `?:` makes sense, it bloats code as less as possible. So I changed my mind. I think that good approach is:
* if variable can initially be `undefined` and after assigning value to it, this variable is never set to `null`/`undefined` in the code then this variable can be defined as `myVariable?: MyType;`, for example:
```
account?: Account;
```
* if variable value can be "reset" in the code then use `myVariable: MyType | null = null;` syntax, for example:
```
account: Account | null = null;
```
and then in the code in the appropriate places assignment can be: `this.account = null;`

I added new commit to follow this path. @wmarques I hope it's OK for you.